code/modules/jobs/job_types/civilian.dm

```diff
@@ -114,6 +114,8 @@ Mime
 		H.mind.AddSpell(new /obj/effect/proc_holder/spell/targeted/mime/speak(null))
 		H.mind.miming = 1
 
+	H.dna.add_mutation(CLOWNMUT)
+	H.dna.add_mutation(WACKY)

```

Comment:
`WACKY` adds the comic sans voice, which I think isn't quite what you intended here.